.github/workflows/ci.yaml

```diff
@@ -0,0 +1,14 @@
+name: CI
+
+on: push
```

Comment:
I believe you need to enable GitHub Actions on the repository somehow before it will start using this workflow.

`on: push` will cause GitHub Actions to run on every pull request change and on `master`.

Reply:
Hi @neverendingqs I believe it's turned on bt default. Is there anything else I need to do?

![image](https://user-images.githubusercontent.com/1049124/98872536-f35fb580-2433-11eb-9057-f145921fa2ff.png)

Reply:
Nope it looks like it's all set up! https://github.com/colynb/serverless-dotenv-plugin/runs/1387663662

You may want to add branch protection to require CI to pass before allowing a pull request to be merged.